### ISSUE TITLE
Audit for a11y.

### ISSUE THREAD

**Issue by lewiscowper** [npm/npm-camp#12]
Self-assigning doesn't work because orgs, but I'll take a look at this one

**Comment by ashleygwilliams** [comment]
:sparkling_heart:

**Comment by lewiscowper** [comment]
so, first up we have the homepage (generated by [a11y](http://npm.im/a11y])):

```
npm.github.io/npm-camp

  ✖ Controls and media elements should have labels

  #field0

  ✖ The web page should have the content's human language indicated in the markup

  html

  ✖ Text elements should have a reasonable contrast ratio

  NAV > .container > A
  NAV > .container > A:nth-of-type(2)
  NAV > .container > A:nth-of-type(3)
  body > .container > .attribution.center > A
  body > .container > .attribution.center > A:nth-of-type(2)
  body > .container > .attribution.center > A:nth-of-type(3)


  ✔ This element does not support ARIA roles, states and properties
  ✔ These elements are focusable but either invisible or obscured by another element
  ✔ Images should have a text alternative or presentational role
  ✔ The purpose of each link should be clear from the link text
  ✔ Meaningful images should not be used in element backgrounds
  ✔ The web page should have a title that describes topic or purpose
```

To get all ✔ here, we'll need:
- [x] Add label to `#field0`. #16 
- [x] Add `lang="en"` to the `<html>` tag #15 
- [ ] Improve contrast in terms of colours on the mentioned elements

Sponsorships Page:

```
npm.github.io/npm-camp/sponsorships.html

  ✖ The web page should have the content's human language indicated in the markup

  html

  ✖ Text elements should have a reasonable contrast ratio

  NAV > .container > A
  NAV > .container > A:nth-of-type(2)
  NAV > .container > A:nth-of-type(3)
  body > .container > SECTION:nth-of-type(3) > .button.button-primary
  body > .container > .attribution.center > A
  body > .container > .attribution.center > A:nth-of-type(2)
  body > .container > .attribution.center > A:nth-of-type(3)


  ✔ This element does not support ARIA roles, states and properties
  ✔ These elements are focusable but either invisible or obscured by another element
  ✔ The purpose of each link should be clear from the link text
  ✔ Meaningful images should not be used in element backgrounds
  ✔ The web page should have a title that describes topic or purpose
```

To get to all ✔ on sponsorships page
- [x] Add `lang="en"` to the `<html>` tag #15 
- [ ] Improve contrast in terms of colours on the mentioned elements

CFP Page

```
npm.github.io/npm-camp/cfp.html

  ✖ The web page should have the content's human language indicated in the markup

  html

  ✖ Text elements should have a reasonable contrast ratio

  NAV > .container > A
  NAV > .container > A:nth-of-type(2)
  NAV > .container > A:nth-of-type(3)
  body > .container > .row > .columns.eight > UL > LI:nth-of-type(5) > A
  body > .container > SECTION:nth-of-type(3) > .button.button-primary
  body > .container > .attribution.center > A
  body > .container > .attribution.center > A:nth-of-type(2)
  body > .container > .attribution.center > A:nth-of-type(3)


  ✔ This element does not support ARIA roles, states and properties
  ✔ These elements are focusable but either invisible or obscured by another element
  ✔ The purpose of each link should be clear from the link text
  ✔ Meaningful images should not be used in element backgrounds
  ✔ The web page should have a title that describes topic or purpose
```

To get all ✔ on the CFP page, we'll need:
- [x] Add `lang="en"` to the `<html>` tag #15 
- [ ] Improve contrast in terms of colours on the mentioned elements

**Comment by lewiscowper** [comment]
The contrast issues are more of a design change, so I guess they'll need some input from @ashleygwilliams. Looks like mostly link colours vs the background colour could be improved in terms of contrast, and the buttons as well. I'm not 100% sure whether the buttons are the internal button colours, or the button background vs background colour.

**Comment by lewiscowper** [comment]
From a readability perspective, the font seems very small, although we should be able to pick that up in the next audit with a different tool. Probably http://wave.webaim.org/ for a closer look at that sort of thing. I've opened #15 and #16 for some quick wins in terms of a11y, but we need to work on contrast + readability for the next pass.
